stock_release_channel_process_end_time/tests/test_release_end_date.py

```diff
@@ -79,12 +78,8 @@ def test_channel_counter_release_ready(self):
         # Asleep the release channel to void the process end date
         self.channel.action_sleep()
         self.channel.invalidate_recordset()
-        self.channel.action_wake_up()
         # Execute the picking channel assignations
-        jobs_after = self.env["queue.job"].search([])
-        for job in jobs_after:
-            job = Job.load(job.env, job.uuid)
-            job.perform()
+        self.channel.with_context(test_queue_job_no_delay=True).action_wake_up()
```

Comment:
'queue_job__no_delay' is the correct key.

See https://github.com/OCA/queue/blob/16.0/queue_job/utils.py#L38